docs/docs-content/clusters/edge/edge-configuration/installer-reference.md

```diff
@@ -228,8 +228,9 @@ The `stylus.site` blocks accept the following parameters.
 | `stylus.site.clusterName`        | The name of the cluster the Edge host belongs to.                                                                                                                                                                                                           | String                           | `''`    |
 | `stylus.site.deviceUIDPaths`     | A list of file paths for reading in a product or board serial that can be used to set the device ID. The default file path is `/sys/class/dmi/id/product_uuid`. Refer to the [Device ID (UID) Parameters](#device-id-uid-parameters) section to learn more. | Array of `FileList`              | None    |
 | `stylus.site.edgeHostToken`      | A token created at the tenant scope that is required for auto registration.                                                                                                                                                                                 | String                           | `''`    |
+| `stylus.site.hostName`           | The host name for the Edge host. This will also be the node's name in Palette when the host is added to a cluster. If you do not specify a host name, the `stylus.site.name` value becomes the host name.                                        | string                           | `''`    |
```

Comment:
```suggestion
| `stylus.site.hostName`           | The host name for the Edge host. This will also be the node's name when the host is added to a cluster. If you do not specify a host name, the `stylus.site.name` value becomes the host name.                                        | string                           | `''`    |
```